packages/suite/src/components/suite/DeviceDisplay.tsx

```diff
@@ -0,0 +1,230 @@
+            return (
+                <Content>
+                    <Chunks isPixelType={isPixelType}>
+                        {showChunksInRows(groupedChunks?.slice(0, 4))}
+                    </Chunks>
+                    <StyledChunkedNextIcon
+                        isPixelType={isPixelType}
+                        size={isPixelType ? 10 : 20}
+                        icon={isPixelType ? 'ADDRESS_PIXEL_NEXT' : 'ADDRESS_NEXT'}
+                        color={isPixelType ? '#ffffff' : '#959596'}
+                    />
+                    <Wrapper>
+                        <Divider isPixelType={isPixelType} />
+                        <AddressLabel isPixelType={isPixelType}>
+                            <Translation id="NEXT_PAGE" />
+                        </AddressLabel>
+                    </Wrapper>
+                    <StyledChunkedContinuesIcon
+                        isPixelType={isPixelType}
+                        size={isPixelType ? 10 : 20}
+                        icon={isPixelType ? 'ADDRESS_PIXEL_CONTINUES' : 'ADDRESS_CONTINUES'}
+                        color={isPixelType ? '#ffffff' : '#959596'}
+                    />
+                    <Chunks isPixelType={isPixelType}>
+                        {showChunksInRows(groupedChunks?.slice(4), true)}
+                    </Chunks>
+                </Content>
+            );
+        }
+
+        return <Chunks isPixelType={isPixelType}>{showChunksInRows(groupedChunks)}</Chunks>;
+    };
+
+    function showOriginal(address: string) {
+        if (isPaginated) {
+            const breakpoint = isPixelType ? 70 : 81;
+            return (
+                <Content>
+                    <Text isPixelType={isPixelType}>{address.slice(0, breakpoint)}</Text>
+                    <StyledNextIcon
+                        isPixelType={isPixelType}
+                        size={isPixelType ? 10 : 20}
+                        icon={isPixelType ? 'ADDRESS_PIXEL_NEXT' : 'ADDRESS_NEXT'}
+                        color={isPixelType ? '#ffffff' : '#959596'}
+                    />
+                    <Wrapper>
+                        <Divider isPixelType={isPixelType} />
+                        <AddressLabel isPixelType={isPixelType}>
+                            <Translation id="NEXT_PAGE" />
+                        </AddressLabel>
+                    </Wrapper>
+                    <StyledContinuesIcon
+                        isPixelType={isPixelType}
+                        size={isPixelType ? 10 : 20}
+                        icon={isPixelType ? 'ADDRESS_PIXEL_CONTINUES' : 'ADDRESS_CONTINUES'}
+                        color={isPixelType ? '#ffffff' : '#959596'}
+                    />
+                    <Text isPixelType={isPixelType}>
+                        &nbsp;&nbsp;&nbsp;{address.slice(breakpoint)}
+                    </Text>
+                </Content>
+            );
+        }
+        return <Text isPixelType={isPixelType}>{address}</Text>;
```

Comment:
`line-height: 2;` for non-segmented text

<img width="322" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/93f0a2d1-4892-465a-a749-db0ab90f2926">

Reply:
I don't think it needed, you don't have it on device like that, @Hermez-cz is okey with the current UI.

Reply:
What I meant is that even if readability is considered OK here:

<img width="322" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/8c6fcc9c-3e2a-40b8-bbcf-803814f1cda0">

line height is different from the segmented version:

<img width="317" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/84d104bb-6e02-4827-b5bd-3dfb34f8d972">

Reply:
That is on purpose. Device also has it like this.